### DISCUSSION TITLE
UNRAID Docker

### DISCUSSION THREAD

**Discussion by benjaminmchale** [philosowaffle/peloton-to-garmin#690] [Q&A]
Hi everyone,
I have been a faithful user of an UNRAID server but I am unable to run this docker on my home server. There are no templates in the Community Applications to ensure the proper paths and variables are configured. Additionally, I cannot find any additional help online regarding this issue. Any help would be greatly appreciated!

**Comment by Ruharc** [comment]
Are you still looking for help on getting this running?  I just got it all setup and can help if you still need it.

**Reply by benjaminmchale** [reply]
@Ruharc I was able to figure it out using docker compose as a command line in the terminal. Were you able to figure out how to use a community template?

**Reply by Ruharc** [reply]
No, I also used the commands from the Linux install page.  Just wanted to make sure!  It would be great if it got added as a Community App though.
